source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
 
-Field        |             Description             |   Type
------------- | :---------------------------------: | -----:
-params       | Geographic area of the subscription | Object
--- iso       |    Country or region information    | Object
----- country |              Iso code               |   Text
----- region  |             Region code             | Number
+### Errors for getting subscriptions
 
-> To create a Subscription, you have to do a POST request with the following body:
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+401            | Unauthorized   | No valid token was provided in the request headers.
+
+## Getting a subscription by id
+
+> Example call for getting a subscription by its id:
 
 ```shell
-curl -X POST https://api.resourcewatch.org/v1/subscriptions \
--H "Authorization: Bearer <your-token>" \
--H "Content-Type: application/json"  -d \
- '{
-   "name": "<name>",
-   "application": "<application>",
-   "language": "<language>",
-   "resource": {
-       "type": "<type>",
-       "content": "<content>"
-   },
-   "datasets" : ["<dataset>"],
-   "env": <environment>,
-   "params": {
-       "iso": {
-           "country": "<iso>",
-           "region": "<region>"
-       }
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>" \
+-H "Authorization: Bearer <your-token>"
+```
+
+> Response:
+
+```json
+{
+  "data": {
+    "type": "subscription",
+    "id": "587cc014f3b3f6280058e478",
+    "attributes": {
+      "name": "Test subscription",
+      "createdAt": "2020-01-16T12:45:08.434Z",
+      "userId": "57a063da096c4eda523e99ae",
+      "resource": {
+        "type": "EMAIL",
+        "content": "example@wri.org"
+      },
+      "datasets": ["viirs-active-fires"],
+      "datasetsQuery": [],
+      "params": {
+        "geostore": "50601ff9257df221e808af427cb47701"
+      },
+      "confirmed": false,
+      "language": "en",
+      "env": "production"
     }
-  }'
+  }
+}
 ```
 
-### From World Database on Protected Areas (wdpa)
+If you know the id of a subscription, then you can access it directly by performing a GET request to the `v1/subscriptions/:id` endpoint. Keep in mind that this will only match subscriptions that are owned by the user who performed the request.
+
+### Errors for getting subscriptions by id
+
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+400            | ID is not valid| The id provided is not a valid subscription id.
+401            | Unauthorized   | No valid token was provided in the request headers.
+404            | Subscription not found | Either no subscription exists with the provided id, or the subscription with the id provided is not owned by the user who performed the request.
```

Comment:
Technically if it's a different user's subscription that should be a 403, right?

Reply:
It should, yes, but the docs are reflecting the way it is currently implemented.

We should add another backlog item to return the correct error response when trying to access subscriptions that don't belong to the logged user.